src/components/Text.js

```diff
@@ -147,7 +148,10 @@ export const H5 = styled.h5`
 `;
 
 H5.defaultProps = {
-  fontSize: '1.2rem',
-  fontWeight: 'bold',
+  fontSize: '2rem',
+  lineHeight: '2.4rem',
+  textAlign: 'center',
```

Comment:
Why is centered text a default prop style? I don't think every H5 used should be centered. Layout should be determined when it is used.